Plugin.php

```diff
@@ -330,6 +336,64 @@ protected function extendWinterPagesPlugin(): void
         }
     }
 
+    /**
+     * Extend the Winter.Sitemap plugin
+     */
+    protected function extendWinterSitemapPlugin(): void
+    {
+        $pluginManager = PluginManager::instance();
+        if (!$pluginManager->exists('Winter.Sitemap')) {
+            return;
+        }
+
+        $typeMapping = [
+            MLPage::class => ['cms-page'],
+        ];
+
+        if ($pluginManager->exists('Winter.Pages')) {
+            $typeMapping[MLPage::class] = array_merge($typeMapping[MLPage::class], ['static-page', 'all-static-pages']);
+        }
+
+        if ($pluginManager->exists('Winter.Blog')) {
+            $typeMapping[MLBlogCategoryModel::class] = ['blog-category', 'all-blog-categories'];
+            $typeMapping[MLBlogPostModel::class] = ['blog-post', 'category-blog-posts', 'all-blog-posts'];
+        }
+
+        Event::listen('winter.sitemap.processMenuItems', function ($item, $url, $theme, $apiResult) use ($typeMapping) {
+            foreach ($typeMapping as $class => $types) {
+                if (in_array($item->type, $types)) {
+                    return $class::resolveMenuItem($item, $url, $theme);
+                }
+            }
+
+            return false;
+        });
+
+        Event::listen('winter.sitemap.makeUrlSet', function ($definition, $xml, $urlSet) {
+            if (Request::has('preview')) {
+                // hack to force browser to properly render the XML sitemap
+                $nsUrl = 'xmlns:xhtml-namespace-definition-URL-here';
+            } else {
+                $nsUrl = 'http://www.w3.org/1999/xhtml';
+            }
+            $urlSet->setAttribute('xmlns:xhtml', $nsUrl);
+        });
```

Comment:
This should be removed and the sitemap plugin should instead implement sitemap stylesheets, which will fix the rendering issues in the browser when the data from the multilanguage sitemaps is present.